controls/cis_rhel7.yml

```diff
@@ -2371,49 +2373,50 @@ controls:
     - gid_passwd_group_same
 
   - id: 6.2.4
-    title: Ensure shadow group is empty (Automated)
+    title: Ensure no duplicate UIDs exist (Automated)
     levels:
     - l1_server
     - l1_workstation
     status: automated
     rules:
-      - ensure_shadow_group_empty
+    - account_unique_id
 
   - id: 6.2.5
-    title: Ensure no duplicate user names exist (Automated)
+    title: Ensure no duplicate GIDs exist (Automated)
     levels:
     - l1_server
     - l1_workstation
     status: automated
     rules:
-    - account_unique_name
+      - group_unique_id
 
   - id: 6.2.6
-    title: Ensure no duplicate group names exist (Automated)
+    title: Ensure no duplicate user names exist (Automated)
     levels:
     - l1_server
     - l1_workstation
     status: automated
     rules:
-      - group_unique_name
+    - account_unique_name
 
   - id: 6.2.7
-    title: Ensure no duplicate UIDs exist (Automated)
+    title: Ensure no duplicate group names exist (Automated)
     levels:
     - l1_server
     - l1_workstation
     status: automated
     rules:
-    - account_unique_id
+      - group_unique_name
 
   - id: 6.2.8
-    title: Ensure no duplicate GIDs exist (Automated)
+    title: Ensure root path integrity (Automated)
     levels:
     - l1_server
     - l1_workstation
     status: automated
     rules:
-      - group_unique_id
+    - accounts_root_path_dirs_no_write
+    - root_path_no_dot
```

Comment:
I think this control should have status set to partial because checks do not cover following cases which are mentioned in the CIS policy:
- Locations that are not directories
- Non root owned directories